README.md

```diff
@@ -1,6 +1,14 @@
+# React Forms Lab
+
 [![Pursuit Logo](https://avatars1.githubusercontent.com/u/5825944?s=200&v=4)](https://pursuit.org)
 
-# [Assignment Title]
+## Functionality
+
+Build a React app containing:
+
+- An input where the user can enter the values in an array
+- A select to choose between calculating the sum, the average, and the [mode](https://www.mathsisfun.com/definitions/mode.html) (most frequently occurring number)
+- A calculate button
```

Comment:
- Can you add a bullet point on what should happen if invalid input is submitted?
- Also what would you think of a markdown table that shows sample inputs and expected outputs? (Including invalid inputs.) This could be taken from the test cases.
- When the solution is done can you add a gif or screenshot?